src/platform/datapath_winkernel.c

```diff
@@ -2195,10 +2195,11 @@ QuicDataPathBindingReturnRecvDatagrams(
         QuicPoolFree(BatchedInternalContext->OwningPool, BatchedInternalContext);
     }
 
-    if (DataIndications != NULL) {
+    if (DataIndications != NULL && Binding != NULL) {
         //
         // Return the datagram indications back to Wsk.
         //
+        QUIC_DBG_ASSERT(Binding != NULL);
```

Comment:
Assert is unnecessary with the check above.

Reply:
Actually, remove the if check instead. Keep the assert.

Reply:
Ah thought I had done that. Whoops. Easy fix.